core/chains/evm/logpoller/log_poller.go

```diff
@@ -291,8 +298,12 @@ func (lp *logPoller) getReplayFromBlock(ctx context.Context, requested int64) (i
 
 func (lp *logPoller) run() {
 	defer close(lp.done)
-	logPollTick := time.After(0)
+	logPollPendingTick := time.After(0)
```

Comment:
Pending is a bit misleading since on first time run we will backfill. I'd leave it as logPollTick and call logPollFinalizedTick as backupPollTick